app/experimenter/targeting/constants.py

```diff
@@ -106,7 +106,32 @@ def __post_init__(self):
     ),
     desktop_telemetry=(
         "{first_run} AND environment.system.os.windows_build_number >= 18362 AND "
-        "!isDefaultBrowser AND doesAppNeedPin"
+        "!isDefaultBrowser AND {new_profile}"
+    ).format(first_run=FIRST_RUN.desktop_telemetry, new_profile=NEW_PROFILE),
+    sticky_required=True,
+    is_first_run_required=False,
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_MOTION = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 needing default and no prefer "
+        "reduced motion"
+    ),
+    slug="first_run_new_profile_need_default_prefers_motion",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default, preferring motion"
+    ),
+    targeting=(
+        "{first_run} && !userPrefersReducedMotion".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
```

Comment:
noticed `first_run= FIRST_RUN.targeting probably` in above targetings, probably ok but will be good to run with @jaredlockhart  if its encouraged to use unique variable name between targetings

Reply:
@punamdahiya can you give me an example of what you mean?

Reply:
@jaredlockhart I think Punam is highlighting that the `first_run` is reused as a format variable amongst multiple configuration declarations, and may be confusing for some.

Reply:
Ah interesting point, I'll think about it 👍